doc/nrf/app_dev/device_guides/fem/fem_nrf2220.rst

```diff
@@ -0,0 +1,149 @@
+.. _ug_radio_fem_nrf2220:
+
+Enabling support for nRF2220
+############################
+
+The nRF2220 device is a range extender that you can use with nRF52, nRF53 and nrf54L Series devices.
+The nRF2220 features an GPIO and I2C interface.
+You can use it to fully control your front-end module.
+To use nRF2220, complete the following steps:
+
+1. Add the following node in the devicetree file:
+
+   .. code-block::
+
+      / {
+            nrf_radio_fem: name_of_fem_node {
+               compatible = "nordic,nrf2220-fem";
+               cs-gpios = <&gpio1 10 GPIO_ACTIVE_HIGH>;
+               md-gpios = <&gpio1 8 GPIO_ACTIVE_HIGH>;
+               twi-if = <&nrf_radio_fem_twi>;
+               output-power-dbm = <10>;
+         };
+      };
+
+#. Optionally replace the device name ``name_of_fem_node``.
+#. Replace the pin numbers provided for each of the required properties:
+
+   * ``cs-gpios`` - GPIO characteristic of the device that controls the ``CS`` signal of the nRF2220.
+   * ``md-gpios`` - GPIO characteristic of the device that controls the ``MD`` signal of the nRF2220.
+
+   These properties correspond to ``CS`` and ``MD`` pins of nRF2220 that are supported by software FEM.
+
+   The ``phandle-array`` type is commonly used for describing GPIO signals in Zephyr's devicetree.
```

Comment:
Are we sure we want to introduce some `phandle-array` name here? It does not appear again and may be confusing.

Reply:
This is similar description to that
https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/app_dev/device_guides/fem/fem_nrf21540_gpio.rst?plain=1#L56
regarding nrf21540

Reply:
It is also described on this doc page here https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/build/dts/phandles.html